.nvmrc

```diff
@@ -1,2 +1 @@
-4
-
+6.9.1
```

Comment:
Moreover `nvm` is not capable of parsing `^6.9.1` so we would be stuck with `6.9` or worse `6`, which could allow versions that we don't want such as 6.8.x and 6.9.0.